src/models/event.ts

```diff
@@ -334,7 +337,10 @@ export class MatrixEvent extends EventEmitter {
         if (this._localRedactionEvent) {
             return {} as T;
         }
-        return (this.clearEvent.content || this.event.content || {}) as T;
+        if (this.clearEvent) {
+           return this.clearEvent.content as T;
```

Comment:
this also needs the `|| {}` bit given content can be `falsey` to match the behaviour applied to the this.event.content below